### ISSUE TITLE
Migrate 🇺🇸 25

### ISSUE THREAD

**Issue by jtemporal** [jtemporal/gitfichas#139]

**Comment by Shivansh-22866** [comment]
Hey, wanted to know if the issue is still active and if the repo is open for contributions?

**Comment by jtemporal** [comment]
@Shivansh-22866 yes and yes! PRs are welcomed! Please take a look at the contributions guidelines and let me know if you have any questions ☺️

**Comment by Shivansh-22866** [comment]
@jtemporal Let me know if there is any changes to be made in the PR
